### PR TITLE
More XML renaming

### PR DESCRIPTION
**Pull request by nrwahl2**
This is the next batch after #3310. Go as far as you want. I've got enough of these stored to last a while.

### CODE REVIEW

File: daemons/controld/controld_cib.c [hunk 1]
```diff
@@ -593,9 +595,11 @@ append_restart_list(lrmd_event_data_t *op, struct ra_metadata_s *metadata,
     }
 
     digest = calculate_operation_digest(restart, version);
-    /* Add "op-force-restart" and "op-restart-digest" to indicate the resource supports reload,
-     * no matter if it actually supports any parameters with unique="1"). */
-    crm_xml_add(update, XML_LRM_ATTR_OP_RESTART,
+    /* Add PCMK__XA_OP_FORCE_RESTART and "op-restart-digest" to indicate the
+     * resource supports reload, no matter if it actually supports any
+     * parameters with unique="1").
```
Comment (kgaillot) [inline daemons/controld/controld_cib.c:600]: while we're here, change that last line to `reloadable parameters`

**Review by kgaillot — COMMENTED** [review]
Reviewed through "Deprecate XML_LOCATION_ATTR_DISCOVERY"

File: daemons/controld/controld_cib.c (hunk 1 above)
Comment (nrwahl2) [reply daemons/controld/controld_cib.c:600]: assuming just `reloadable parameters` and not `reloadable parameters with unique="1"`

Comment (nrwahl2) [reply daemons/controld/controld_cib.c]: done

**Review by kgaillot — APPROVED** [review]

**Pull request merged**